tutorials/MD/03_MD_Setup_Production_Run.ipynb

```diff
@@ -0,0 +1,274 @@
+    "\n",
+    "`% ssh -X username@oscgate.arc.ox.ac.uk`\n",
+    "\n",
+    "When prompted, enter your password and hit Enter.\n",
+    "Now, you need to connect to the arcus-htc cluster, so in your command line type:\n",
+    "\n",
+    "`% ssh -X arcus-htc`\n",
+    "\n",
+    "If you get a message like this one:\n",
+    "\n",
+    "The authenticity of host 'arcus-htc (10.137.128.21)' can't be established.\n",
+    "RSA key fingerprint is d1:83:21:b1:f2:bd:8f:e7:5d:cd:74:d1:73:b9:70:7a.\n",
+    "Are you sure you want to continue connecting (yes/no)? \n",
+    "\n",
+    "Type \n",
+    "\n",
+    "`% yes`\n",
+    "\n",
+    "You can now check the path to the directory you are at, by typing:\n",
+    "\n",
+    "`% pwd`\n",
+    "\n",
+    "It should be your home directory (/home/username). This is where you will work from, you will need to create some subdirectories to organise your files. First creatge a directory for your setup and move to it, as it will be your first working directory:\n",
+    "\n",
+    "`% mkdir setup`\\\n",
+    "`% cd setup`\n",
+    "\n",
+    "Now, you need to load GROMACS, which is the simulation software package that you will utilise:\n",
+    "\n",
+    "`% module load gpu/gromacs/2020.1`\n",
+    "\n",
+    "Check that it was successfully loaded, by typing:\n",
+    "\n",
+    "`% gmx help commands`\n",
+    "\n",
+    "This will print basic information for every built-in GROMACS command. If you need more detailed information on a command in particular, you can type:\n",
+    "\n",
+    "`% gmx [command] -h`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Specifics to GROMACS\n",
+    "\n",
+    "GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles. (http://www.gromacs.org/)\n",
+    "\n",
+    "- Coordinates files have the extension .gro and the default name is conf.gro.\n",
+    "- The topology file (default name topol.top) contains all the information about which atoms are bonded to which and what force-field parameters are applied etc.\n",
+    "- The trajectory files have the extension .xtc and .trr, the former does not contain velocity information and coordinates are held at a reduced precision, and so occupies less disk space. However you will need velocities if you want to continue a simulation.\n",
+    "- The .edr file contains the energy information from the trajectory.\n",
+    "- The .mdp file contains the information that was used to setup the actual simulation. Things related to temperature, pressure, how the electrostatics is calculated etc. Although we will not use this today, the file is provided so you can see how this trajectory was made.\n",
+    "- The .ndx file allows you to specify atoms or groups of atoms for use in analysis, restraints, etc and is optional.\n",
```

Comment:
Do we generate ndx files anywhere in this tutorial?

Reply:
No, you are right. We did in the previous version for HB analysis. I deleted it.